### DISCUSSION TITLE
In chat rooms, drag username bar fully to the right (Hide it)

### DISCUSSION THREAD

**Discussion by zenalv** [nicotine-plus/nicotine-plus#2780] [Q&A]
A user posted this once, as shown in the video, he drags the username bar with all the names fully to the right which hides it.
Me using win11 can not do this on 3.2.9. It stops and I can not drag and hide it all the way.
What is the secret?
https://github.com/nicotine-plus/nicotine-plus/assets/89178880/2704e5fc-c04f-429d-91bd-fc34feefb4c3

**Comment by slook** [comment]
There is a minimum width set. In the latest unstable version, the panel is not manually adjustable at all.

At the minimum Main Window width, the Speed and Files columns are unimportant and perhaps they could be hidden.

**Comment by mathiascode** [comment]
If there are valid reasons for hiding the sidebar, perhaps a button for this could be added at some point. I've noticed that e.g. Discord has one.

Could you tell us why you want to hide the user list?

**Reply by zenalv** [reply]
I wish to make the interface as minimal as possible. When resizing the window to as small as possible, the sidebar takes up 40% of the window. Also I am obviously jealous and I still want to know the secret of the user in the video.
Also I know who I am chatting with and don't need to constantly check who it is I'm chatting with. I simply don't care where they come from, how many files they share, what speed they have, therefore I do not need that window being seen all the time.
Also I find it wrong that it won't be able to even move the sidebar in future updates.
back to qt

**Reply by slook** [reply]
I wish to avoid lines wrapping whilst being able to use a larger font in the chat text view.
